drivers/dma/dma_stm32.c

```diff
@@ -84,19 +103,26 @@ static void dma_stm32_irq_handler(void *arg)
 	if (func_ll_is_active_tc[id](dma)) {
 		func_ll_clear_tc[id](dma);
 		stream->dma_callback(stream->callback_arg, id + STREAM_OFFSET,
-					0);
+			0);
```

Comment:
To be moved to commit: `drivers: dma: configuration of the dma_stm32`
But yet the alignment is not correct, this could come from you editor setting (1 tab equals 8 spaces)

Reply:
moved to commit` drivers: dma: channel configuration for dma_stm32`